src/testdrive/src/action/kafka/ingest.rs

```diff
@@ -182,6 +209,9 @@ pub fn build_ingest(mut cmd: BuiltinCommand) -> Result<IngestAction, String> {
             Some(Format::Protobuf {
                 descriptor_file,
                 message,
+                confluent_wire_format: cmd.args.opt_bool("confluent-wire-format")?.unwrap_or(false),
+                schema_id: cmd.args.opt_parse::<i32>("schema-id")?.unwrap_or(885),
+                schema_message_id: cmd.args.opt_parse::<u8>("schema-message-id")?.unwrap_or(0),
```

Comment:
These should all be prefixed with `key-`; otherwise they'll always be false because extracting the options above will have already consumed them. (Calling `opt_parse` removes the argument from the map.)